edgebuilder-install.sh

```diff
@@ -468,28 +468,18 @@ install_cli_rpm()
 uninstall_server()
 {
     export DEBIAN_FRONTEND=noninteractive
+    # check if edgebuilder-server is currently installed
     if dpkg -s edgebuilder-server; then
 
-        sudo rm -rf /opt/edgebuilder/server/vault
-        # attempt autoremove
-        if sudo apt autoremove -qq edgebuilder-server -y ;then
-            echo "Successfully autoremoved server components"
+        sudo rm -rf /opt/edgebuilder/server
```

Comment:
this should be /opt/edgebuilder we we dont expect the server and node to be installed together